thread/src/components/Common/CommonPassword.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
CommonPassword and CommonField are exactly the same.

Remove this, and in CommonField do the following.

```
const CommonField = ({onChangeText, value, label, style, secureTextEntry = false}) => (
  <Item style={style} stackedLabel>
      <TextStyle>{label}</TextStyle>
      <InputStyle
          onChangeText={onChangeText}
          value={value}
          secureTextEntry={secureTextEntry}
      />
  </Item>
);

```